### PR TITLE
Release notes for week of 5 December 2016

### PR DESCRIPTION
**Pull request by srpearce**
## [DOC-3512](https://openedx.atlassian.net/browse/DOC-3512)

Add a description of your changes with links to any relevant material.

### Date Needed (optional)

6 December 2016

### Reviewers

Possible roles follow. The PR submitter checks the boxes after each reviewer finishes and gives :+1:. 

- [x] Subject matter expert: @cptvitamin (item in the studio_2016-12-05.rst file)
- [x] Subject matter expert: @efischer19 (second item in the lms_2016-12-05.rst file)
- [x] Doc team review (sanity check, copy edit, or dev edit?): @catong 
- [ ] Product review: @sstack22 (both items in the lms_2016-12-05.rst file)
- [ ] Partner support: 
- [ ] PM review: 

### Testing

- [ ] Ran ./run_tests.sh without warnings or errors

### Post-review

- [ ] Add a comment with the description of this change or link this PR to the next release notes task.
- [ ] Squash commits

### CODE REVIEW

File: en_us/release_notes/source/2016/lms/lms_2016-12-05.rst [hunk 1]
```diff
@@ -0,0 +1,10 @@
+* The grade report and the problem grade report now include "Not Attempted" and
+  "Not Accessible" values for assignments or problems that learners have not
+  attempted or did not have access to, respectively. For more information, see
+  :ref:`Access_grades`. (:jira:`TNL-5577`)
+
+* Permissions for the Staff role have been updated. Course team members with
+  the Staff role can now access and modify grades for individual learners.
+  Course team members with the Admin role can access and modify grades for all
+  learners in a course. For more information, see :ref:`Administrative Team
+  Roles`. (:jira:`TNL-5907`)
```
Comment (efischer19) [inline en_us/release_notes/source/2016/lms/lms_2016-12-05.rst:10]: This seems functionally correct to me, I'd just ask @catong to confirm the phrasing we're using - I remember there being some confusion about what the docs claimed with regards to the student admin tab of the instructor dashboard.

So 👍 from me, but @catong can suggest changes if needed.

Comment (catong) [reply en_us/release_notes/source/2016/lms/lms_2016-12-05.rst:10]: @srpearce @efischer19 "Staff" and "Admin" roles are consistent with our descriptions in the "Building and Running" guide. Are there any finer distinctions w.r.t. reset or grade adjustment abilities other than being able to reset/adjust grades for a single learner vs all learners in the course?

Comment (efischer19) [reply en_us/release_notes/source/2016/lms/lms_2016-12-05.rst:10]: No, what you've described here captures what happens in code quite clearly, there aren't any finer distinctions to be made that I know of.

Thanks!

**Comment by cptvitamin** [comment]
👍

**Comment by sstack22** [comment]
👍

File: en_us/release_notes/source/2016/lms/lms_2016-12-05.rst (hunk 1 above)
Comment (catong) [reply en_us/release_notes/source/2016/lms/lms_2016-12-05.rst:10]: Just want to verify with @sstack22 that we want to characterize this bug fix as an "update" to permissions?

**Comment by catong** [comment]
👍 with one question for @sstack22 re calling the permissions bug fix a "permissions update".

**Pull request merged**